app/helpers/application_helper/toolbar/container_group_center.rb

```diff
@@ -58,6 +58,11 @@ class ApplicationHelper::Toolbar::ContainerGroupCenter < ApplicationHelper::Tool
           'pficon pficon-edit fa-lg',
           N_('Edit Tags for this #{ui_lookup(:table=>"container_group")}'),
           N_('Edit Tags')),
+        button(
+          :container_group_protect,
+          'pficon pficon-edit fa-lg',
+          N_('Manage Policies for this #{ui_lookup(:table=>"container_group")}'),
+          N_('Manage Policies')),
```

Comment:
I think we need two new buttons in all of these classes: one for manage policies and one to run compliance test.

This applies to all of the X_center.rb files